base/_buttons.scss

```diff
@@ -0,0 +1,63 @@
+@charset "UTF-8";
+
+// buttons.scss
+// This file contains all styles related to the button element.
+// see http://bitters.bourbon.io
+// dependent on utilities/color.scss
+// -----------------------------------------------------------------------------
+
+
+// Variables in use on this file.
+// Variables are defined with !default flags, and can be overwritten in
+// `utilities/_variables.scss`
+// -----------------------------------------------------------------------------
+$action-color: color(generic, blue) !default;
+$reverse-font-color: color(generic, white) !default;
```

Comment:
Each of these will need inline comments. I am unsure what reverse-font-color is or why I would need it.